code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -182,30 +182,6 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 // DRIVERS //
 /////////////
 
-/datum/clockwork_scripture/channeled/belligerent //Belligerent: Channeled for up to fiteen times over thirty seconds. Forces non-servants that can hear the chant to walk. Nar-Sian cultists are burned.
```

Comment:
Yeah okay I guess, I still want to have something named belligerent for this slot, though. Maybe this could do some damage?